lib/make/numbers.ts

```diff
@@ -39,13 +39,12 @@ export class numbers {
    * @param bypass - Determines if the range should be exceeded.
    */
   static int8(bypass = false): number {
-    if (bypass || random.chance(50)) {
-      return numbers._exceed(
-        random.choose([
-          [1, -128],
-          [10, 127],
-        ])
-      )
+    if (bypass && random.chance(50)) {
```

Comment:
I agree this makes more sense (`&&` vs `||`), but maybe it should be < 50? previously it was 100% if bypass was true.

Reply:
50 is a fairly low chance here in this case but the general idea is that we still want to prefer the numbers generated following it rather than exceeding the bounds.